### PR TITLE
fix  paginator

### PR DESCRIPTION
**Pull request by casterweb**

### CODE REVIEW

File: src/app/models/writing-test/writing-test.component.scss [hunk 1]
```diff
@@ -14,9 +14,10 @@ form {
   padding: 1rem;
 }
 
-.description {
+p {
   margin-bottom: 1rem;
   text-align: center;
```
Comment (shuliakleonid) [inline src/app/models/writing-test/writing-test.component.scss:19]: You get style on all tag p it`s correct?

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

File: src/app/models/writing-test/writing-test.component.scss (hunk 1 above)
Comment (casterweb) [reply src/app/models/writing-test/writing-test.component.scss:19]: this is an attempt to fix the styles, but then the server was turned off

**Review by shuliakleonid — APPROVED** [review]

**Review by shuliakleonid — APPROVED** [review]

**Review by ExxZzyy — APPROVED** [review]

File: src/app/models/grammar-test/grammar-test.component.scss [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
   margin-bottom: 1rem;
   text-align: center;
 }
 
 .grammar-stepper__position {
   display: block;
 }
+
+mat-card {
+  margin: 50px 30px 100px 30px;
```
Comment (tsiananel) [inline src/app/models/grammar-test/grammar-test.component.scss:11]: Let's use rem and short definition

File: src/app/models/listening-test/listening-test.component.scss [hunk 3]
```diff
@@ -9,6 +9,10 @@
   margin: 0 auto;
 }
 
+mat-card {
+  margin: 50px 30px 100px 30px;
```
Comment (tsiananel) [inline src/app/models/listening-test/listening-test.component.scss:13]: Rem & shorthand definition

**Review by tsiananel — APPROVED** [review]

**Pull request merged**